Gathondu/Amity
# Ft add tests 1

#### What does this PR do?
* Adds tests for the amity project
* Tests include the tests for amity and room
 
#### Description of Task to be completed?
* Add tests for the functionality of the modeled classes

#### How should this be manually tested?
* Make sure you have `nosetest` installed and run the following command in the terminal pointing to your amity project folder. `nosetests --with-coverage tests/test_room.py`

      N.B test_amity still to be worked on
